### ISSUE TITLE
help in class_weights.npy file

### ISSUE THREAD

**Issue by divyadaiict** [mdciri/Vox2Vox#8]
Hello, I am unable to understand the contents of class_weights.npy file. My dataset is 3d mr images having 7 classes to classify of heart region . I know something similar you have mentioned in #2 but where to mention the names of my labels and size etc. could you please explain it in detail, thanks.

**Comment by mdciri** [comment]
Dear Divyadaiict,

the class weight are those weights calculated from the training label
dataset and they tell to the model "how much to focus" on that class.
For example, in image segmentation the pixels (or voxel if 3D case) that
belong to the background are normally many, but that is not the class that
you want your model to detect most.

Hence you assign a weight to each class you want to segment to balance the
model training. Here
<https://scikit-learn.org/stable/modules/generated/sklearn.utils.class_weight.compute_class_weight.html>
there is a simple way to calculate them.



Best regards,
Marco Domenico Cirillo

Il giorno ven 25 giu 2021 alle ore 13:07 divyadaiict <
***@***.***> ha scritto:
